Student/SubmitAssignment/Assignments/AssignmentsViewController.swift

```diff
@@ -41,47 +41,34 @@ class AssignmentsViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        title = NSLocalizedString("Assignments", bundle: .core, comment: "")
-        tableView.dataSource = self
-        tableView.delegate = self
-        tableView.backgroundColor = .clear
-        view.backgroundColor = .clear
+        title = NSLocalizedString("Assignments", comment: "")
```

Comment:
These strings should not be looked up from .core, since there is a Localizable.strings file in this project that the i18n team is translating.

Also the tableView setup is happening in the storyboard.